cloud/cmd/inter_cluster_gateway/main.go

```diff
@@ -110,7 +118,7 @@ func main() {
 	version := pcap.Version()
 	fmt.Println(version)
```

Comment:
remove this print if it's for debugging, or else output using logger

Reply:
Removed